lib/game.js

```diff
@@ -1,10 +1,15 @@
+/* eslint-env node */
+
+const uuidV4 = require('uuid/v4');
+
 const { GameEngine } = require('./engine');
 
 
 class Game {
   constructor(server) {
+    this.id = uuidV4();
     this.server = server;
-    this.players = [];
+    this.players = {};
```

Comment:
Maps vs. objects. What the [MDN says](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map):
>  Use maps over objects when keys are unknown until run time, and when all keys are the same type and all values are the same type.
>
> Use objects when there is logic that operates on individual elements.